lib/matobj.gi

```diff
@@ -1568,3 +1568,142 @@ InstallMethod( \[\,\]\:\=, "for a matrix object, two positions, and an object",
   end );
 
 
+############################################################################
+# Elementary matrix operations
+############################################################################
+
+InstallMethod( MultMatrixRow, "for a mutable matrix object, one row index, and a scalar",
```

Comment:
This may just be me, but following up on my previous comment, I think I would find it easier if this said `MultMatrixRowLeft`, i.e., 
```suggestion
InstallMethod( MultMatrixRowLeft, "for a mutable matrix object, one row index, and a scalar",
```
That way it is immediate that the code is correct, I don't need to remember that `MultMatrixRow` is a synonym.

But of course this is subjective, feel free to disagree, this PR shall not be blocked over this :-)